### PR TITLE
Lipstick

### PR DESCRIPTION
**Pull request by AlbertSu123**

### CODE REVIEW

File: src/actions/Swap.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ const getOneInchApproveTx = async (tokenAddress: string, amt: number, options: E
     return transaction as TransactionData
 }
 
-const getOneInchSwapTx = async (swapParams: SwapParams, fromAddress: string, options: EnvOption) => {
+const getOneInchSwapTx = async (swapParams: SwapParams, fromAddress: string, options: EnvOption): Promise<any> => {
```
Comment (csirak1528) [inline src/actions/Swap.ts:27]: shouldnt this be txparams

**Review by csirak1528 — CHANGES_REQUESTED** [review]

File: src/actions/Swap.ts (hunk 1 above)
Comment (AlbertSu123) [reply src/actions/Swap.ts:27]: I don't think so, this is returned from the 1 inch api so I have no idea what type this is. Why do you think it is txParams?

Comment (csirak1528) [reply src/actions/Swap.ts:27]: from the functinon name it should conform to the tx params type but its okay

**Review by csirak1528 — APPROVED** [review]

File: src/actions/Swap.ts (hunk 1 above)
Comment (AlbertSu123) [reply src/actions/Swap.ts:27]: Unfortunately I don't think that is the case

**Review by Chazzzzzzz — APPROVED** [review]

**Pull request merged**